SPECS/rpmdevtools/rpmdevtools.spec

```diff
@@ -104,103 +116,59 @@ echo %%{_datadir}/bash-completion > %{name}.files
 [ -d %{buildroot}%{_sysconfdir}/bash_completion.d ] && \
 echo %%{_sysconfdir}/bash_completion.d > %{name}.files
 
+%if ! 0%{?azl}
 for dir in %{_emacs_sitestartdir} ; do
   install -dm 755 %{buildroot}$dir
   ln -s %{_datadir}/rpmdevtools/rpmdev-init.el %{buildroot}$dir
   touch %{buildroot}$dir/rpmdev-init.elc
 done
+%endif
 
 # For backwards compatibility
 ln -sr %{buildroot}%{_bindir}/rpmdev-spectool %{buildroot}%{_bindir}/spectool
 echo ".so man1/rpmdev-spectool.1" > %{buildroot}%{_mandir}/man1/spectool.1
 
-%if 0%{?rhel}
+%if 0%{?rhel} || 0%{?azl}
 cp %{SOURCE1} %{buildroot}%{_datadir}/rpmdevtools/
 %py_byte_compile %{python3} %{buildroot}%{_datadir}/rpmdevtools/
 %endif
 
+%check
+check_ok=true
```

Comment:
why do the check_ok thing here instead of just failing on the first tool that returns nonzero? I don't understand what the benefit of checking *all* the tools is; once we know that any one of them fails then the build should just stop, since some fix is needed...doesn't seem any point to me to continue checking the rest?

Reply:
%check is dumb... it only "returns" the last command's exit code, but `exit 1` causes `rpmbuild` itself to fail which means the check section itself doesn't actually complete.

Reply:
well, right, %check fails on the first error...what i'm saying is i don't see the point of not using the normal behavior of simply failing on the first error, instead of keeping track of `check_ok`